src/styles/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-top: 16px;
+  margin-right: 3px;
+  margin-left: 3px;
```

Comment:
A better practice is to write these 3 properties at one `margin` attribute.

[More info](https://developer.mozilla.org/ru/docs/Web/CSS/margin)